scripts/benchmarks/benches/velocity.py

```diff
@@ -9,12 +9,11 @@
 from utils.utils import run, create_build_path
 from .options import options
 import os
-import re
 
 class VelocityBench:
     def __init__(self, directory):
         self.directory = directory
-        self.repo_path = git_clone(self.directory, "velocity-bench-repo", "https://github.com/oneapi-src/Velocity-Bench", "34ee4ebe18d91dfdd38b7d798fd986b41874fcbc")
+        self.repo_path = git_clone(self.directory, "velocity-bench-repo", "https://github.com/mateuszpn/Velocity-Bench", "66e8188")
```

Comment:
your PR was merged:
https://github.com/oneapi-src/Velocity-Bench/commit/b22215c16f789100449c34bf4eaa3fb178983d69

Can you update?

Otherwise lgtm.

Reply:
Done

Reply:
no? I still see your fork being used.

Reply:
ok, I see the new push.